app/logic/userManagement.py

```diff
@@ -33,3 +33,27 @@ def changeCurrentTerm(term):
     newCurrentTerm.save()
 
     session["current_term"] = model_to_dict(newCurrentTerm)
+
+def addTerm():
+    terms_table = {"Spring":"Summer",
+                    "Summer":"Fall",
+                    "Fall":"Spring"}
+    terms = list(Term.select().order_by(Term.id))
+    lastCreatedTerm = terms[-1]
+    termDescription = lastCreatedTerm.description
+    term = termDescription.split()
+    year = termDescription[-1]
+    if termDescription[0]=="Fall": # We only change the year when it is Fall
+        year = int(year) + 1
+
+    createdTermDescription = terms_table[term[0]]+" "+str(year)
+    academicYear = lastCreatedTerm.academicYear
+    if termDescription[0] == "Summer": #we only change academic year when the latest term in the table is Summer
+        previousAY = academicYear.split("-")
+        academicYear = str(int(previousAY[0])+1)+"-"+str(int(previousAY[-1])+1)
+
+    isSummer = False
+    if "Summer" in createdTermDescription.split():
+        isSummer = True
```

Comment:
you can just assign the boolean expression to the `isSummer` variable, without having to use an `if` at all

Reply:
We did not get this well. Do you mean shortening it to: 

`isSummer = True if "Summer" in createdTermDescription.split() else False`

Reply:
No. You are giving a boolean expression to the if statement, just use the same expression for the variable.
`isSummer = "Summer" in createdTermDescription.split()`

Reply:
fixed